packages/react-storage/src/components/FileUploader/ui/FileList/__tests__/__snapshots__/FileList.test.tsx.snap

```diff
@@ -0,0 +1,324 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FileList renders an alert in case of error 1`] = `
+<div>
+  <div
+    class="amplify-storagemanager__file__list"
+  >
+    <div
+      class="amplify-storagemanager__file"
+    >
+      <div
+        class="amplify-storagemanager__file__wrapper"
+      >
+        <div
+          class="amplify-storagemanager__file__main"
+        >
+          <p
+            class="amplify-text amplify-storagemanager__file__name"
```

Comment:
Looks like it's still using the old classnames?

Reply:
Updated to use name referencing FileUploader